Resources/Prototypes/Entities/Markers/Spawners/Random/salvage.yml

```diff
@@ -49,6 +49,25 @@
         - state: red
         - sprite: Objects/Weapons/Melee/crusher.rsi
           state: icon
+    - type: RandomSpawner
```

Comment:
Do not indent the listed components (and layers) since they already start with ` - `. They should be aligned, like the prototypes below

Reply:
That's a good start, but mainly I meant the entire component block